### ISSUE TITLE
rollTables is not iterable

### ISSUE THREAD

**Issue by thmarth** [spetzel2020/quick-encounters#121]
Using version 1.1.1

Quick Encounters is broken if you don't have any roll tables defined in the encounter.
Tested this with both a QE created before the update and a new QE,

```
QuickEncounter.js:1133 Uncaught (in promise) TypeError: rollTables is not iterable
[Detected 1 package: quick-encounters]
    at QuickEncounter.run (QuickEncounter.js:1133:33)
```

```
        //1.1.1 If we have rollTables, then roll them to generate additional extractedActors which we add temporarily 
        // (but only to extractedActors not to the property)
        for (const rollTable of rollTables) {
            const rollTableObject = game.tables.get(rollTable.rollTableId);

```
Also tested downgrading to 1.1.0 and encounters are working again

**Comment by spetzel2020** [comment]
Yikes. Will fix (and test better!) immediately.

**Comment by spetzel2020** [comment]
Released in 1.1.2
